dashboard/app/views/levels/_blockly.html.haml

```diff
@@ -18,5 +19,4 @@
     appOptions.locale = '#{js_locale}';
     //]]>
 
-  %script{src: '/shared/js/client_api.js'}
```

Comment:
The portions of client_api that we need get bundled into our initApp.js. Not sure if it would be less confusing if our generated bundle had a different name.